terraform.tf.j2

```diff
@@ -12,38 +12,20 @@ terraform {
 }
 
 provider "aws" {
-  version = "~> 3.12.0"
+  version = "~> 3.42.0"
   region  = "{{terraform.provider_region}}"
 
   assume_role {
     role_arn = "arn:aws:iam::${lookup(local.account, local.environment)}:role/${var.assume_role}"
   }
-}
-
-locals {
-  name        = "dataworks-repo-template-terraform"
-  environment = terraform.workspace == "default" ? "development" : terraform.workspace
-
-  account = {
-    {%- for key, value in accounts.items() %}
-      {{key}} = "{{value}}"
-    {%- endfor %}
-  }
 
-  common_tags = {
-    {%- for key, value in common_tags.items() %}
-      {{key}} = "{{value}}"
-    {%- endfor %}
-    Environment  = local.environment
-    Application  = local.name
-    Team         = "DataWorks"
+  default_tags {
+    tags = local.common_repo_tags
   }
+}
 
-  cidr_block = {
-  {%- for environment, ranges in cidr_block.items() %}
-      {{ environment }} = {
-        {%- for key, value in ranges.items() %}
-          {{ key }} = "{{ value }}"{% endfor %}
-      } {%- endfor %}
-  }
```

Comment:
Why drop this?

Reply:
I've just moved it to locals file @ChrisScottThomas with the other locals